website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md

```diff
@@ -2,8 +2,9 @@
 title: How are TikTok, Flipkart, KubeSphere, and others using OpenEBS for Local Volumes
 author: Kiran Mova
 author_info: Contributor and Maintainer OpenEBS projects. Chief Architect MayaData. Kiran leads overall architecture & is responsible for architecting, solution design & customer adoption of OpenEBS.
-tags: solutions tutorials chaosengineering
+tags: Localpv, Openebs, Flipkart, TikTok, Kubernetes, Mayastor, Mayadata
```

Comment:
```suggestion
tags: Localpv, Openebs, Flipkart, TikTok, Kubernetes, Mayastor, MayaData
```

Reply:
also can we change --> 

Openebs ==>  OpenEBS

Reply:
Done